src/consts/parameters.ts

```diff
@@ -5,10 +5,19 @@
 export const contractConst = "0x436492DBc2E30E56FaC8F2297BD1964833c0687d";
 
 // The name of the chain your contract is deployed to.
-// Provide a string for standard chains, or import one from @thirdweb-dev/chains package.
-// Learn more: https://portal.thirdweb.com/react/react.thirdwebprovider#activechain-recommended
+// For standard chains. (e.g. "mainnet", "rinkeby", "polygon", "mumbai", "base")
+// For custom chains, provide the chain name from thirdweb chain list or use the import name from @thirdweb-dev/chains.
+// Read: https://portal.thirdweb.com/react/react.thirdwebprovider#activechain-recommended
+// Chainlist: https://thirdweb.com/chains
+// e.g."{Sepolia}" or "{Gnosis}"
```

Comment:
Don't need the `{` `}` here

Reply:
It doesn't work without `{``}` if it is custom chain.

Reply:
I'm not saying it should be a string - `"Sepolia"`, I'm saying it should be an object - `Sepolia`

Example: 

```ts
export const chainConst = Sepolia;
```

Reply:
I see, thanks for the correction!